examples/embeddable_explorer/public/hello_world_embeddable_example.tsx

```diff
@@ -54,27 +51,27 @@ export function HelloWorldEmbeddableExample({ getEmbeddableFactory }: Props) {
           <EuiText>
             Here the embeddable is rendered without the factory. A developer may use this method if
             they want to statically embed a single embeddable into their application or page.
+            `input` may be used to declaratively update current embeddable input
           </EuiText>
           <EuiPanel data-test-subj="helloWorldEmbeddablePanel" paddingSize="none" role="figure">
-            <EmbeddableRoot embeddable={new HelloWorldEmbeddable({ id: 'hello' })} />
+            <EmbeddableRenderer
```

Comment:
`EmbeddableRenderer` this new component allows to pass either `embeddable` or `factory`. types make sure it is one of.